content/learn/other-resources.md

```diff
@@ -39,13 +39,24 @@ IIIF (International Image Interoperability Framework) is an emerging standard fo
 - [IIF Awesome](https://github.com/IIIF/awesome-iiif)
 - [IIIF Apps & Demos](https://iiif.io/apps-demos/#image-servers)
 
+## Large File Storage
+
+- [Git Large File Storage](https://git-lfs.github.com/)
+
 ## Markdown
 
 - [Markdown Cheatsheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf)
 - [10-minute Markdown Tutorial](https://commonmark.org/help/)
 - [Mastering Markdown](https://guides.github.com/features/mastering-markdown/)
 - [The Programming Historian: Getting Started with Markdown](https://programminghistorian.org/en/lessons/getting-started-with-markdown)
 
+## Netlify
+
+With Netlify, you can build a quick preview site or link it to your Github account to automatically generate an updated preview every time you push changes to your project. You can also use it to host your final project when it’s ready to publish. 
+
+- [Netlify Documentation](https://docs.netlify.com/?utm_medium=email&_hsmi=90174540&_hsenc=p2ANqtz-9n3XbaCpnuVnhAyXWNnU_uaKGJiH9tDe0mHei6hnuBrx87Cblx6xfQyMgkoklqEw9xBplyoRaii6-GNIB7sx5UwcEyvw&utm_content=90174540&utm_source=hs_automation)
```

Comment:
Be careful about using links with a lot of extra crud on them. These come from copy and pasting from various sources are usually related to tracking, telling Netlify or other services they use where the link came from. We should always use the simplest, cleanest version. In this case: https://docs.netlify.com/

Reply:
Ah, yes, this is a bad habit. I'll be sure to keep it clean moving forward. Just fixed it!